{"start": 1582384615718, "befores": [{"status": "broken", "start": 1582384615718, "stop": 1582384615721, "name": "setup", "statusDetails": {"message": "TypeError: super() takes at least 1 argument (0 given)\n", "trace": "  File \"/usr/local/lib/python2.7/dist-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python2.7/dist-packages/_pytest/fixtures.py\", line 936, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python2.7/dist-packages/_pytest/fixtures.py\", line 791, in call_fixture_func\n    res = next(it)\n  File \"/home/azg/Documents/Projekt_grupowy/pydqz1-baltycki-python/automated-tests/conftest.py\", line 11, in setup\n    driver = webdriver.Chrome(ChromeDriverManager().install())\n  File \"/usr/local/lib/python2.7/dist-packages/webdriver_manager/chrome.py\", line 24, in install\n    driver_path = self.download_driver(self.driver)\n  File \"/usr/local/lib/python2.7/dist-packages/webdriver_manager/manager.py\", line 34, in download_driver\n    cached_path = self.driver_cache.find_file_if_exists(driver.get_name(), driver.get_os_type(),\n  File \"/usr/local/lib/python2.7/dist-packages/webdriver_manager/driver.py\", line 48, in get_os_type\n    if \"win\" in super().get_os_type():\n"}}], "stop": 1582384615836, "uuid": "6dabbe5f-d71c-4bbf-b6b1-eba31f138799", "children": ["b1e14322-b362-420c-89d4-a48e142d0bd8"]}